packages/python-packages/api-stub-generator/apistub/nodes/_function_node.py

```diff
@@ -98,6 +98,9 @@ def _parse_function(self):
         # Find signature to find positional args and return type
         sig = inspect.signature(self.obj)
         params = sig.parameters
+        # Add all keyword only args here temporarily until docstring is parsed
+        # This is to habdle the scenario is keyword arg typehint (py2 style is present in signature itself)
```

Comment:
Typo: handle
And it's Py3 style where the type hints are in the signature itself